src/components/SurveyQuestion/index.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render } from '@testing-library/react';
+
+import SurveyQuestion from '.';
+import { SurveyQuestion as SurveyQuestionInterface } from 'services/surveyInterfaces';
+
+import { build } from '@support/factory';
+
+describe('SurveyQuestion', () => {
+  describe('given unsupported type', () => {
+    it('renders unsupported notice', () => {
+      const question = build('surveyQuestion', {
+        displayType: 'awesomeTextarea',
+      }) as SurveyQuestionInterface;
+
+      const { getByText } = render(<SurveyQuestion question={question} />);
```

Comment:
In other places, you're using `screen.getByText` to get the element, should we do the same at here?

Reply:
Updated in [b9663ef](https://github.com/hanam1ni/react-surveyor/pull/78/commits/b9663ef1d1b59b478c3c1935175a0001c035edb4)